modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -3,12 +3,17 @@
 // Module included in the following assemblies:
 //
 // * networking/metallb/metallb-configure-address-pools.adoc
-[id="nw-metallb-addresspool-cr_{context}"]
-= About the address pool custom resource
+[id="nw-metallb-ipaddresspool-cr_{context}"]
+= About the IPAddressPool custom resource
 
-The fields for the address pool custom resource are described in the following table.
+[NOTE]
+====
+The address pool custom resource defintion (CRD) and API documented in link:https://docs.openshift.com/container-platform/4.10/networking/metallb/metallb-configure-address-pools.html[About the address pool custom resource] can still be used in {product-version}. However the enhanced functionality associated with advertising the IP address pools with layer 2 or the BGP protocol is not supprted when using the address pool CRD.   
```

Comment:
I would avoid linking to our own doc set like this. Remember that these docs are also on the Customer Portal, so users there will be redirected to docs.openshift.com, which can be confusing. I know that this is likely a workaround for not being able to include xrefs in modules right now, but please consider removing the link.